src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
```

Comment:
inversedRobot name will be more meaningful
```suggestion
  const inversedRobot = {};
```